### ISSUE TITLE
suggestion: Background jobs

### ISSUE THREAD

**Issue by afifurrohman-id** [afifurrohman-id/tempsy#32]
# Description
> Helpful for purge anonymous user and delete file

## Strategy
Have a jobs run consistently each week (or some day) using Github Actions that purge expired file.
If user didn't even hit, endpoint (so middleware for purging not trigger) this would be helpful.

**Comment by harshRn** [comment]
Hi @afifurrohman-id 
I think I can pick this up, although I don't understand why it has been marked as a bug.If I am missing something please do elaborate on the context, otherwise I will start reading, understanding and adding code. 
Thanks!

**Comment by afifurrohman-id** [comment]
Yeah because i just implement the middleware that it's bug when app is block during some request, also slow down response time.

I will also plan this to split up the repo btw, maybe bad for now, because is just small app it's super overkill haha, but i like to experiment.

I will create this using rust btw.
